### PR TITLE
Restore appearance of green reblog symbol in ER

### LINKED ISSUE #736: ER breaks tumblr green reblogs
> oh! okay. did that. and upon doing that, noticed another thing. the new editable reblogs thing stops the little reblog button from glowing green after you've reblogged a post so it shows 'hey i already reblogged this thing'. i was about to send an ask about that but then i noticed it back again after turning the ERs off

probably going to be somewhat difficult to do this the right way. maybe we need to be triggering a different event?

### PR DESCRIPTION
**Pull request by blackjackkent**
Resolves #736

### CODE REVIEW

File: Extensions/editable_reblogs.js [hunk 1]
```diff
@@ -355,10 +355,21 @@ XKit.extensions.editable_reblogs = new Object({
...
+					var id = location_items[1];
 					XKit.interface.kitty.set(response.getResponseHeader("X-tumblr-kittens"));
 					XKit.tools.add_function(function() {
 						Tumblr.Events.trigger("postForms:saved");
-					}, true, "");
+						var windowPost = window.Tumblr.Posts.find({
+							id: parseInt(add_tag)
+						});
+						windowPost.updateReblogControl();
+						Tumblr.Events.trigger("post:reblog:set", {
```
Comment (nightpool) [inline Extensions/editable_reblogs.js]: what if its an edit, does that work too?

Comment (nightpool) [inline Extensions/editable_reblogs.js]: or will it just make you look like you've reblogged the post, when you havent?

Comment (blackjackkent) [inline Extensions/editable_reblogs.js]: Excellent point. I will tweak this later.

**Comment by blackjackkent** [comment]
Per @nightpool 's comment, need to update this so that it only does the green arrow when the post is a) a reblog and b) being posted. (also verify how this works in vanilla tumblr if queueing/drafting, duplicate if necessary)

**Comment by ThePsionic** [comment]
@blackjackkent When drafting, queueing or scheduling, the button still turns green in vanilla Tumblr.

**Pull request closed without merging**